docs/docs/api/multiplayer.md

```diff
@@ -96,23 +96,53 @@ Rune.initLogic({
 })
 ```
 
-## `Rune.gameOver()`
+## `Rune.gameOver(options)`
 
-When the game has ended (with a winner or draw), the action handler should call `Rune.gameOver()`. Your game doesn't need to show a "game over" screen. Rune overlays a standardized high score interface to the user.
+When the game has ended, the action handler should call `Rune.gameOver`. Your
```

Comment:
Are these line breaks yours or some auto-formatter (the other lines are unaffected, so assuming yours). Usually I prefer to not line break markdown files like this as it makes diffs for even small changes difficult to read.

Reply:
For me it's harder to view/edit text in the editor if I have to constantly scroll horizontally back and forth

Reply:
What editor do you use? Almost all of them have a setting to enable word wrap.

Reply:
Didn't know about it, thanks for mentioning this

Reply:
Removed the hard wraps https://github.com/rune/rune-games-sdk/pull/128/commits/bf03ee5e26a674f6d4db3e48945787f5fd478b3a